web/docs/guides/database/extensions/pg-cron.mdx

```diff
@@ -0,0 +1,109 @@
+---
+id: pgcron
+title: "pg_cron: Postgres Schedular"
+description: An HTTP Client for PostgreSQL Functions.
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+The `pg-cron` extension allows you to schedule job for PostgreSQL.
+## Overview
+
+Let's cover some basic concepts:
+
+- pg_corn:pg_cron is a simple cron-based job scheduler for PostgreSQL (10 or higher) that runs inside the database as an extension.
+  - `schedule` − Schedules a job to be executed during a certain span of time.
+  - `unschedule` − Stops a scheduling of a job.
+  - `alter_job` − Function to change job properties.
+  - `schedule_in_database` - Function to schedule in a custom database.
+  - `log_min_messages` - Setting to control log_min_messages in pg_corn launcher.
+  - `enable_superuser_jobs` - Setting to disallow superuser jobs.
+
+You can use the `pg_cron` extension to make these network requests from Postgres.
```

Comment:
I don't know about the "network requests" - it's really just for managing cron jobs, and has nothing to do with networking